empty_things3_inbox_to_notion.py

```diff
@@ -28,17 +28,22 @@ def addParagraphToBlock(block_id, paragraph_content):
 
 
 def addContentToBlock(
-    block_id, content: list, *, padded=True, blank_header=False, as_callouts=False
+    block_id, content: list, *, title="", padded=True, blank_header=False, as_type: tn.BlockTypes=None
 ):
-    if as_callouts:
-        content_ = [tn.create_callout_block(title="", children=content)]
+    if as_type == tn.BlockTypes.CALLOUT:
+        content_ = [tn.create_callout_block(title=title, children=content)]
+    elif as_type == tn.BlockTypes.TOGGLE:
+        content_ = [tn.create_toggle_block(title=title, children=content)]
+    elif as_type:
+        content_ = [tn.create_block(title=title, children=content, type=type)]
     elif padded:
         content_ = [tn.create_paragraph("")] + content + [tn.create_paragraph("")]
         if blank_header and content and not tn.objIsHeader(content[0]):
```

Comment:
todo later: 

- refactor padding option
- unify all the create block options into one function